### PR TITLE
Styling changes when 'uses_encryption' is not determined

### LINKED ISSUE #2578: Uses Encryption criteria when unknown should be sad face ☹️
If `Uses encryption` is `unknown` then the site should show the red sad face (and red highlight) instead of the gray neutral face.

screen shot 2019-02-04 at 10 57 41 am

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #2578

### CODE REVIEW

**Comment by kristinashu** [comment]
Tested here https://foundation-mofostaging-pr-2633.herokuapp.com/en/privacynotincluded/products/establish-thing/ and looks right!

**Comment by mmmavis** [comment]
@Pomax 👀 ?

File: network-api/networkapi/buyersguide/templates/fragments/product-criterion.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="criterion criterion-{{ value|lower }} criterion-{% if value == bad %}bad{% else %}good{% endif %}">
+<div class="criterion criterion-{{ value|lower }} {{criterion}} criterion-{% if value == bad %}bad{% else %}good{% endif %}">
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/templates/fragments/product-criterion.html]:
```suggestion
<div class="criterion criterion-{{ value|lower }} {{criterion-name}} criterion-{% if value == bad %}bad{% else %}good{% endif %}">
```

File: network-api/networkapi/buyersguide/templates/product_page.html [hunk 2]
```diff
@@ -97,7 +97,7 @@ <h3 class="h3-heading h3-heading-small">Can it spy on me</h3>
         <h3 class="h3-heading h3-heading-small">What does it know about me</h3>
 
         <div class="mb-5">
-          {% include "fragments/product-criterion.html"  with  value=product.uses_encryption  bad=False  help=product.uses_encryption_helptext  label="Product uses encryption" %}
+          {% include "fragments/product-criterion.html"  with  value=product.uses_encryption  bad=False  help=product.uses_encryption_helptext  label="Product uses encryption" criterion="uses-encryption" %}
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/templates/product_page.html]:
```suggestion
          {% include "fragments/product-criterion.html"  with  value=product.uses_encryption  bad=False  help=product.uses_encryption_helptext  label="Product uses encryption" criterion-name="uses-encryption" %}
```

**Comment by Pomax** [comment]
R+ if we rename that variable to `criterion-name`, to keep the code that tiny bit more readable

File: network-api/networkapi/buyersguide/templates/fragments/product-criterion.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<div class="criterion criterion-{{ value|lower }} criterion-{% if value == bad %}bad{% else %}good{% endif %}">
+<div class="criterion criterion-{{ value|lower }} {{criterion-name}} criterion-{% if value == bad %}bad{% else %}good{% endif %}">
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/templates/fragments/product-criterion.html]: yay, python

```suggestion
<div class="criterion criterion-{{ value|lower }} {{criterion_name}} criterion-{% if value == bad %}bad{% else %}good{% endif %}">
```

File: network-api/networkapi/buyersguide/templates/product_page.html [hunk 4]
```diff
@@ -97,7 +97,7 @@ <h3 class="h3-heading h3-heading-small">Can it spy on me</h3>
         <h3 class="h3-heading h3-heading-small">What does it know about me</h3>
 
         <div class="mb-5">
-          {% include "fragments/product-criterion.html"  with  value=product.uses_encryption  bad=False  help=product.uses_encryption_helptext  label="Product uses encryption" %}
+          {% include "fragments/product-criterion.html"  with  value=product.uses_encryption  bad=False  help=product.uses_encryption_helptext  label="Product uses encryption" criterion-name="uses-encryption" %}
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/templates/product_page.html]:
```suggestion
          {% include "fragments/product-criterion.html"  with  value=product.uses_encryption  bad=False  help=product.uses_encryption_helptext  label="Product uses encryption" criterion_name="uses-encryption" %}
```

**Comment by mmmavis** [comment]
🎉

**Pull request merged**